src/scry/symbol.cr

```diff
@@ -91,13 +91,20 @@ module Scry
 
       line_number = location.line_number
       column_number = location.column_number
-
       position = Position.new(line_number - 1, column_number - 1)
-      range = Range.new(position, position)
-      location = Location.new(@document_uri, range)
+      end_location = node.end_location
+      if end_location
```

Comment:
you can do `if end_location = node.end_location`, I think it shows better that `end_location` won't be used outside the if